src/Interpolation.jl

```diff
@@ -244,13 +244,14 @@ Download or recompute interpolation coefficients.
 """
 function interpolation_setup(;Γ=NamedTuple(),
         lon=[i for i=-179.:2.0:179., j=-89.:2.0:89.],
-        lat=[j for i=-179.:2.0:179., j=-89.:2.0:89.])
+        lat=[j for i=-179.:2.0:179., j=-89.:2.0:89.],
+        path=tempname())
```

Comment:
- Could you change this to `filename=tempname()*"_interp_coeffs.jld2")` ?

Otherwise it's kind of confusing what path is (neither folder nor file name)

- also please add docstring for the new parameter